packages/active-campaign-client/src/__tests__/handlers/manageListSubscription.test.ts

```diff
@@ -0,0 +1,26 @@
+import { manageListSubscription } from '../../handlers/manageListSubscription';
+
+// remove .skip to run the test, be aware it does real API calls
+describe('manage list subscription', () => {
+  const cognitoUserId = '66ae52a0-f051-7080-04a1-465b3a4f44cc';
```

Comment:
what is the proper name of this constant cognitoId, cognitoUserId or username?

Reply:
maybe for both this and the next const we should add a proper env variable and a dedicated test env to properly set or create a user and a list before running test :thinking: